erpnext/accounts/report/accounts_receivable/accounts_receivable.html

```diff
@@ -19,7 +19,7 @@
 
 
 <h2 class="text-center">{%= __(report.report_name) %}</h2>
-<h4 class="text-center">{%= filters.customer || filters.supplier %} </h4>
+<h4 class="text-center">{%= report.context_data.customer_name || filters.supplier %}</h4>
```

Comment:
if filters.customer then `(report.context_data.customer_name || filters.customer)`
And the same is applicable for supplier as well in accounts_payable report